net/nginx/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nginx
 PKG_VERSION:=1.16.1
-PKG_RELEASE:=1
+PKG_RELEASE:=3
```

Comment:
Any reason to skip `2`?

Reply:
yes there is another pr to be merged before this